sdk/qumulo/Azure.ResourceManager.Qumulo/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("Qumulo")]
```

Comment:
This  should be name of ARM resource, I guess it's `Qumulo.Storage` ?

Reply:
@ArthurMa1978 yes arm resource name is like this "acctest8090/providers/Qumulo.Storage/fileSystems/nikhgup-test-1". do i need to just change the name in this file or update the namespace/folder structure of project as well